src/controllers/ArtsController.js

```diff
@@ -141,7 +145,7 @@ class ArtsController {
 
       if (artistId !== artToUpdate.artistId) {
         const response = new Response(
-          'Not Ok',
+          'Forbidden',
           403,
           'Unauthorized to Edit Article',
           {}
```

Comment:
I don't think passing an empty object here is quite necessary on this error response. Why not remove the empty object.

Reply:
Ok thanks